lib/commands/skill/add-locales/ui.js

```diff
@@ -0,0 +1,40 @@
+const fs = require('fs');
+const inquirer = require('inquirer');
+const path = require('path');
+
+const Messenger = require('@src/view/messenger');
+
+module.exports = {
+    selectLocales,
+    showSourcesForEachLocale
+};
+
+function selectLocales(localeChoices, callback) {
+    inquirer.prompt([{
+        message: 'Please select at least one locale to add:',
+        type: 'checkbox',
+        name: 'localeList',
+        choices: localeChoices
+    }]).then((answer) => {
+        callback(null, answer.localeList);
+    }).catch((error) => {
+        callback(error);
+    });
+}
+
+/**
+ * Display the result of locales addition
+ * @param {Map} iModelSourceByLocales Map { locale: filePath/remoteURI }
+ */
+function showSourcesForEachLocale(iModelSourceByLocales) {
+    Messenger.getInstance().info('The following skill locale(s) have been added according to your local project:');
```

Comment:
if 0 added , it still shows this message

Reply:
Ah I see, I will address this case better